### ISSUE TITLE
NiceJourney crash

### ISSUE THREAD

**Issue by MajorCooke** [TazTxUK/MinimapAPI#86]
Whenever I make a new game now and I press and hold tab, if I release it, instant crash.

```
[INFO] - Saving PersistentGameData to Steam Cloud: rep_persistentgamedata1.dat.
[INFO] - [warn] item pool ran out of repicks
[INFO] - music stopped playing
[INFO] - Lua stack trace:
[INFO] - [C](-1): __add
[INFO] - ...s/minimapi_1978904635/scripts/minimapapi/nicejourney.lua(110): ?
[INFO] - Caught exception, writing minidump...
```

**Comment by namishere** [comment]
Workshop release isn't caught up with github yet.. try again with the copy on here

**Comment by wofsauge** [comment]
Thanks for the report.
it hopefully got fixed now with commit [d208edf](https://github.com/TazTxUK/MinimapAPI/commit/d208edf323c78f0a836d565e9ac3d7c86db5affc)

hopefully the crashes dont occur anymore :)
